### ISSUE TITLE
Global balancing loss

### ISSUE THREAD

**Issue by AmericaBG** [ngessert/patch-lesion#1]
Hi!
Firstly, thank you ver much for your code! It's a great work!

I would like to ask you for help with global dice loss in keras-tensorflow.

Have you implemented it? Or could you tell me where in the files you have shared can I have a look?

Thank you very much!
